payment-service/src/main/java/com/hedvig/paymentservice/serviceIntergration/memberService/MemberServiceImpl.kt

```diff
@@ -48,6 +49,33 @@ class MemberServiceImpl(
     }.map { it.id }
   }
 
+    override fun hasMemberConnectedPaymentForMarket(memberId: String, market: Market): Boolean {
```

Comment:
Since we have payin and payout methods, its better to be specific that we want to get the payin method
```suggestion
    override fun hasMemberConnectedPayinMethodfForMarket(memberId: String, market: Market): Boolean {
```